tests/LogsActivityTest.php

```diff
@@ -42,7 +42,7 @@ public function it_will_log_the_creation_of_the_model()
 
         $this->assertInstanceOf(get_class($this->article), $this->getLastActivity()->subject);
         $this->assertEquals($article->id, $this->getLastActivity()->subject->id);
-        $this->assertEquals('created', $this->getLastActivity()->description);
+        $this->assertEquals('created', $this->getLastActivity()->event);
```

Comment:
The `description` is still the event by default.
So please add the new `event` assertion instead of replacing the old one.